Tasks/PublishToAzureServiceBus/task.json

```diff
@@ -50,7 +50,6 @@
     }
   },
   {
-    "name": "signPayload",
```

Comment:
shouldn't we have name of the property, it helps us uniquely identify it in case need be arises.